### PR TITLE
teams fx samples migration

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

File: samples/tab-app-monetization/nodejs/ClientApp/teamsapp.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Visit https://aka.ms/teamsfx-v5.0-guide for details on this file
+# Visit https://aka.ms/teamsfx-actions for details on actions
+version: 1.0.0
+
+environmentFolderPath: ./env
+projectId: 34174a0f-535b-4d8a-af94-e3e60326e88e
```
Comment (formulahendry) [inline samples/tab-app-monetization/nodejs/ClientApp/teamsapp.yml:6]: remove projectId

**Review by formulahendry — APPROVED** [review]

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Comment by ChetanSharma-msft** [comment]
@Prithvi-MSFT - Please run build pipeline before merging the PR.

**Review by Harikrishnan-MSFT — APPROVED** [review]
Approved

File: samples/tab-app-monetization/nodejs/ClientApp/teamsapp.yml (hunk 1 above)
Comment (Prithvi-MSFT) [reply samples/tab-app-monetization/nodejs/ClientApp/teamsapp.yml]: removed

**Comment by Prithvi-MSFT** [comment]
Build status - All builds passed
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5688535018

**Pull request merged**